src/pages/home/HeaderView.js

```diff
@@ -192,7 +192,7 @@ function HeaderView(props) {
                                 {!_.isEmpty(parentNavigationSubtitleData) && (
                                     <ParentNavigationSubtitle
                                         parentNavigationSubtitleData={parentNavigationSubtitleData}
-                                        parentReportID={props.report.parentReportID}
+                                        parentReportID={`${props.report.parentReportID}`}
```

Comment:
Please revert

Reply:
This shows console warning because `parentReportID` is a number

https://github.com/Expensify/App/blob/33963292487e00e42f2b4d5c0413a1f647a8ff4d/src/components/ParentNavigationSubtitle.js#L21

Reply:
Let me revert it and update the proptypes

Reply:
This is supposed to be a string. I think we have another problem (not related to this PR)

Reply:
done